monorepo-convention-plugins/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+plugins {
+    base
+    alias(libs.plugins.sonarqube)
+    alias(libs.plugins.kotlinx.kover)
+}
+
+version = "0.0.1" // x-release-please-version
+
+// Duplicating functionality of project convention plugin because it is not available in the monorepo-convention-plugins project.
+val lint by tasks.registering { }
+
+val qualityCheck by tasks.registering { }
+
+val lintAll by tasks.registering {
+    dependsOn(lint)
+}
+
+val checkAll by tasks.registering {
+    dependsOn(subprojects.map { "${it.name}:checkAll" })
+}
+
+val qualityCheckAll by tasks.registering {
+    dependsOn(checkAll)
+    dependsOn(qualityCheck)
+}
+
+val buildAll by tasks.registering { dependsOn(qualityCheckAll) }
```

Comment:
> Consider refactoring the common functionality into a shared plugin or script that can be applied to both this project and the main project convention plugin. This would reduce duplication and improve maintainability.

Refactoring is not possible due to a chicken and egg problem

> The checkAll task depends on subprojects.map { "${it.name}:checkAll" }. This assumes that all subprojects have a checkAll task. It might be more robust to filter subprojects that actually have this task.

It is assumed that all projects in the monorepo will have the `checkAl`l` task defined